Content.Server/GameObjects/Components/GUI/HandsComponent.cs

```diff
@@ -208,7 +221,7 @@ private bool DroppedInteraction(ItemComponent item, bool doMobChecks)
             return true;
         }
 
-        public bool TryHand(IEntity entity, [MaybeNullWhen(false)] out string handName)
+        public bool TryHand(IEntity entity, [MaybeNullWhen(false)] out string? handName)
```

Comment:
This change is not really necessary at least with Rider, as the MaybeNullWhen(false) annotation is enough and this leads to having to suppress (incorrect) nullable warnings with the ! operator elsewhere when this method returns true.

```suggestion
        public bool TryHand(IEntity entity, [MaybeNullWhen(false)] out string handName)
```